addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/ConsolePanel.java

```diff
@@ -447,10 +447,6 @@ public void setScript(ScriptWrapper script) {
             // so only do for text based ones
             setTabFocus();
         }
-
-        if (!isTabVisible()) {
-            setTabFocus();
-        }
```

Comment:
We shouldn't just remove it, it should be optional. In any case it's better if this change is done in a separate PR.

Reply:
okay sure